dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -24,12 +24,14 @@ def clever
 
   # GET /users/auth/google_oauth2/callback
   def google_oauth2
+    user = find_user_by_credential
+    user&.update_oauth_credential_tokens auth_hash
```

Comment:
I snuck this fix in as well: We were only calling `update_oauth_credential_tokens` when users sign in, but we also want to update a user's tokens after they have given us Google Classroom access. Now, any time a user hits this callback and is logged in, we will update their tokens.